pyanaconda/modules/storage/storage.py

```diff
@@ -263,10 +235,10 @@ def scan_devices_with_task(self):
         storage = self.storage.copy()
 
         # Set up the storage.
-        storage.ignored_disks = self._disk_selection_module.ignored_disks
-        storage.exclusive_disks = self._disk_selection_module.exclusive_disks
-        storage.protected_devices = self._disk_selection_module.protected_devices
-        storage.disk_images = self._disk_selection_module.disk_images
+        storage.ignored_disks = self._modules["disk_selection"].ignored_disks
```

Comment:
I am worried that using a dictionary to get a module means that pylint or IDE won't be able to recognize incorrect attributes or hint the right ones. Would it be possible to use the manager just for managing the list of modules, but keep the instances for the direct access? You could drop the names in that case.

Reply:
Yes, I think that's the better way to do this, too. About the names, they make some sense, but perhaps let's not treat them as a "primary key", but rather property of the managed module(s). Then, everything still has a name.